packages/core/src/repositories/documentVersionsRepository/index.ts

```diff
@@ -252,4 +252,16 @@ export class DocumentVersionsRepository extends Repository<
 
     return Result.ok(documentsFromMergedCommits)
   }
+
+  async getDocumentsForImport(projectId: number) {
+    const documents = await this.db
+      .selectDistinct({
+        documentUuid: this.scope.documentUuid,
+        path: this.scope.path,
+      })
+      .from(this.scope)
+      .where(eq(this.scope.projectId, projectId))
+
+    return Result.ok(documents)
+  }
```

Comment:
What if the document has been deleted? This list could get annoyingly long on long-lived projects

Reply:
A nice problem to have, though

Reply:
💯 good point!